libsolutil/CMakeLists.txt

```diff
@@ -33,6 +33,8 @@ set(sources
 	SwarmHash.h
 	UTF8.cpp
 	UTF8.h
+	Uri.cpp
```

Comment:
I think this should be URI, just like UTF8 as it is an abbreviation?

Reply:
right. I'll touch this file in about a minute anyways (for using RE instead).